packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.component.js

```diff
@@ -39,21 +42,20 @@ function FieldTemplate(props) {
 
 if (process.env.NODE_ENV !== 'production') {
 	FieldTemplate.propTypes = {
-		children: PropTypes.oneOf(
-			PropTypes.element,
-			PropTypes.arrayOf(PropTypes.element)
-		),
+		children: PropTypes.oneOf(PropTypes.element, PropTypes.arrayOf(PropTypes.element)),
```

Comment:
This is not du to your changes but `PropTypes.oneOf` accepts an array

Reply:
and it's oneOfType